bot.py

```
@@ -0,0 +1,610 @@
+        test = read()
+        final = ""
+        for key in test:
+            #if type(name) == "NoneType":
+            final += "<@{}>".format(key) + " is a " + players[key].getType().split(".")[1].replace("'>", "") + "\n"
+        await message.channel.send(final)
+
+    if msg[0] == ".stats":
+        s = "<@{}>".format(message.author.id)
+        await message.channel.send(_raccoonArray[message.author.id].get_stats(s))
+
+    if msg[0] == ".join" and not message.author.id in fighterIDs:
```

Ruff (0.14.0)

Test for membership should be `not in`

Convert to `not in`

(E713)